test/Intl/Basics.js

```diff
@@ -12,11 +12,20 @@ function testRangeError(tag) {
 }
 
 function assertEachIsOneOf(actualList, expectedList, msg) {
+    if (actualList.length === 0) {
```

Comment:
wondering if this method checks if `actualList` contains any duplicates? Do we care about it?

Reply:
It's true that there should be no duplicates, but that wasn't the intent of this function. This test is intended to be the basics only. i.e. does each API work at all as intended. Things like no duplicates are checked in other tests.